RetailCoder.VBE/Inspections/InspectionsUI.resx

```diff
@@ -522,4 +522,40 @@
   <data name="ObjectVariableNotSetInspectionName" xml:space="preserve">
     <value>Object variable assignment requires 'Set' keyword</value>
   </data>
+  <data name="DefaultProjectNameInspectionResultFormat" xml:space="preserve">
+    <value>Project '{0}' has default name</value>
+  </data>
+  <data name="ObsoleteCallStatementInspectionResultFormat" xml:space="preserve">
+    <value>Assignment uses obsolete 'Call' modifier</value>
+  </data>
+  <data name="ObsoleteCommentSyntaxInspectionResultFormat" xml:space="preserve">
+    <value>Comment uses obsolete 'Rem' marker</value>
+  </data>
+  <data name="ObsoleteLetStatementInspectionResultFormat" xml:space="preserve">
+    <value>Assignment uses obsolete 'Let' modifier</value>
+  </data>
+  <data name="ImplicitActiveSheetReferenceInspectionResultFormat" xml:space="preserve">
+    <value>Member '{0}' implicitly references ActiveSheet</value>
+  </data>
+  <data name="ImplicitActiveWorkbookReferenceInspectionResultFormat" xml:space="preserve">
+    <value>Member '{0}' implicitly references ActiveWorkbook</value>
+  </data>
+  <data name="MultipleDeclarationsInspectionResultFormat" xml:space="preserve">
+    <value>Instruction contains multiple declarations</value>
+  </data>
+  <data name="ObsoleteTypeHintInspectionResultFormat" xml:space="preserve">
+    <value>{0} of {1} '{2}' uses an obsolete type hint</value>
+  </data>
+  <data name="OptionBaseInspectionResultFormat" xml:space="preserve">
+    <value>Component '{0}' uses 'Option Base 1'</value>
+  </data>
+  <data name="String1" xml:space="preserve">
```

Comment:
That is the default value; it appears I accidentally tricked it into adding it to the thing.